include/deal.II/fe/fe.h

```diff
@@ -2767,6 +2767,16 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
    * it is no longer necessary.
    */
   virtual std::unique_ptr<InternalDataBase>
+  get_face_data(const UpdateFlags               update_flags,
+                const Mapping<dim, spacedim> &  mapping,
+                const hp::QCollection<dim - 1> &quadrature,
+                dealii::internal::FEValuesImplementation::
+                  FiniteElementRelatedData<dim, spacedim> &output_data) const;
```

Comment:
@kronbichler I have adjusted the other functions also that there are two version. However, noticed an issue with `FiniteElement::get_face_data`, since it already has an implementation in the base class so that the version taking `Quadrature` will never be called. Any suggestions!?